### PR TITLE
Faqs fix

### PR DESCRIPTION
**Pull request by scrummish**

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -153,5 +153,8 @@ module.exports = {
     // this (optional) plugin enables Progressive Web App + Offline functionality
     // To learn more, visit: https://gatsby.dev/offline
     // `gatsby-plugin-offline`,
+    // require("dotenv").config({
```
Comment (mbwatson) [inline gatsby-config.js:156]: probably don't need this

File: package.json [hunk 2]
```diff
@@ -5,6 +5,10 @@
   "version": "0.1.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
+    "@emotion/react": "^11.9.0",
```
Comment (mbwatson) [inline package.json:8]: once we sort this all out, we'll need to remember to remove these dependencies

File: src/hooks/use-freshdesk-faqs.js [hunk 3]
```diff
@@ -30,27 +29,33 @@ export const useFreshdeskFaqs = category => {
...
+              .then((response) => {
                 // A Freshdesk article's status is either 1 or 2, for draft or published, respectively
                 // We'll filter here for only the published articles.
                 folder.articles = response.data.filter(
-                  article => article.status === 2
+                  (article) => article.status === 2
                 );
                 return folder;
               })
+              .then((result) => {
```
Comment (mbwatson) [inline src/hooks/use-freshdesk-faqs.js:49]: what's going on here? why are we passing in result but not using it?

**Review by mbwatson — COMMENTED** [review]
looks good. i left a couple comments for change. and i had one question about the hook functionality

File: src/hooks/use-freshdesk-faqs.js (hunk 3 above)
Comment (scrummish) [reply src/hooks/use-freshdesk-faqs.js]: Left over from testing other strategies, removed now.

**Pull request merged**